.github/workflows/pr.yml

```diff
@@ -64,3 +63,26 @@ jobs:
             ajv validate -s ../text/simple_extensions_schema.yaml --strict=true --spec=draft2020 -d "$i"
           done
         working-directory: ./extensions
+  commitlint:
```

Comment:
it would be great to document the rules for future commiters and add a pre commit hook (so they aren't surprised by failed ci and have to re-write their commit). I am happy to take that as a follow up if you want.